app/actions/remote/groups.ts

```diff
@@ -4,13 +4,13 @@
 import {Client} from '@client/rest';
 import NetworkManager from '@managers/network_manager';
 
-export const getGroupsForAutocomplete = async (serverUrl: string, channelId: string) => {
+export const getGroupsForAutocomplete = async (serverUrl: string, query: string) => {
     let client: Client;
     try {
         client = NetworkManager.getClient(serverUrl);
     } catch (error) {
         return [];
     }
 
-    return client.getAllGroupsAssociatedToChannel(channelId, true);
+    return client.getGroups(query);
```

Comment:
Yeah, in the webApp it lives as `searchGroups( ...`, but it's still a `GET` request to the root `/groups?q=...` endpoint with a slightly different query param.

Server code is [here](https://github.com/mattermost/mattermost-server/blob/856f9b11af5f12856969749934df56f1d4fcbdc3/api4/group.go#L96) and lightly out of date docs [here](https://api.mattermost.com/#operation/GetGroups)

Reply:
Yeah, the webapp makes a GET request to `/api/v4/groups?q=&filter_allow_reference=true&page=0&per_page=60&include_member_count=true`. Note that the rest request here in the mobile app omits `include_member_count=true`, but I think that makes sense because we're not displaying the group member count. I do think we should in the future, but that's out of scope of the work we're doing here.